### PR TITLE
Added Bears vs Babies

### LINKED ISSUE #56: Add Bears v.s Babies
Add Bears v.s Babies to the list

### PR DESCRIPTION
**Pull request by CoreyT355**
Fixes #56

Submitting this PR means you have:

- [X] searched the [readme][readme] file and searched the [open pull requests](https://github.com/edm00se/awesome-board-games/pulls) (no duplicates please)
- [X] and will follow [the desired format](https://github.com/edm00se/awesome-board-games/blob/master/formatting.md)
- [X] used image links that resolve correctly, are appropriately sized, and are served over HTTPS

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -403,6 +403,15 @@
...
 
+### [Bears vs. Babies](https://boardgamegeek.com/boardgame/211534/bears-vs-babies)
+
+> Bears vs Babies is a card game in which you build handsome, incredible monsters who go to war with horrible, awful babies. The shared deck of cards consists of bear parts (and other monster parts) and baby cards. When you draw a part, you use it to build a monster for yourself; when you draw a baby, it goes in the center of the table. When babies are provoked, they attack, and anyone who has fewer monster parts than the number of attacking babies loses their monster; everyone with more parts than babies defeats this infantile army and scores.
+
+![Bears vs. Babies Image](https://cf.geekdo-images.com/imagepage/img/mCG6F7WCV0iC0bnXa_onRxeYx2w=/fit-in/900x600/filters:no_upscale()/pic3231155.png)
+
+| Players | Min. Age | Time    |
+| ------: | -------: | ------: |
+|  2 - 5  |        7 |     15 - 20m |
```
Comment (edm00se) [inline readme.md]: The [Travis CI checks failed](https://travis-ci.org/edm00se/awesome-board-games/builds/592045799#L228-L233) the formatting requirement. The formatting for the table cell of "time" should look more like this:

```suggestion
|  2 - 5  |        7 | 15 - 20m |
```

File: readme.md [hunk 2]
```diff
@@ -403,6 +403,15 @@
 | ------: | -------: | ------: |
 |  1 - 4  |       13 | 90-120m |
 
+### [Bears vs. Babies](https://boardgamegeek.com/boardgame/211534/bears-vs-babies)
+
+> Bears vs Babies is a card game in which you build handsome, incredible monsters who go to war with horrible, awful babies. The shared deck of cards consists of bear parts (and other monster parts) and baby cards. When you draw a part, you use it to build a monster for yourself; when you draw a baby, it goes in the center of the table. When babies are provoked, they attack, and anyone who has fewer monster parts than the number of attacking babies loses their monster; everyone with more parts than babies defeats this infantile army and scores.
+
+![Bears vs. Babies Image](https://cf.geekdo-images.com/imagepage/img/mCG6F7WCV0iC0bnXa_onRxeYx2w=/fit-in/900x600/filters:no_upscale()/pic3231155.png)
```
Comment (edm00se) [inline readme.md]: The image is incredibly large compared to every other image and the target size from the formatting guide. I found a slightly better one on the Bears vs Babies website. Updated, this line would look like this:

```suggestion
![Bears vs. Babies Image](https://explodingkittens.com/img/store/games/bvb_core_thumb_x1.png)
```

**Comment by edm00se** [comment]
Refs: #55

File: readme.md (hunk 2 above)
Comment (CoreyT355) [reply readme.md]: Ah that is a much better one. Updated.

File: readme.md (hunk 1 above)
Comment (CoreyT355) [reply readme.md]: Fixed

File: readme.md [hunk 3]
```diff
@@ -403,6 +403,15 @@
...
 
+### [Bears vs. Babies](https://boardgamegeek.com/boardgame/211534/bears-vs-babies)
+
+> Bears vs Babies is a card game in which you build handsome, incredible monsters who go to war with horrible, awful babies. The shared deck of cards consists of bear parts (and other monster parts) and baby cards. When you draw a part, you use it to build a monster for yourself; when you draw a baby, it goes in the center of the table. When babies are provoked, they attack, and anyone who has fewer monster parts than the number of attacking babies loses their monster; everyone with more parts than babies defeats this infantile army and scores.
+
+![Bears vs. Babies Image](https://explodingkittens.com/img/store/games/bvb_core_thumb_x1.png)
+
+| Players | Min. Age | Time    |
+| ------: | -------: | ------: |
+|  2 - 5  |        7 |15 - 20m |
```
Comment (edm00se) [inline readme.md]: It looks like we lost some of the padding, which will cause the linter to fail. It needs a minimum of a single space for padding in the last cell.

Comment (CoreyT355) [reply readme.md]: My bad, still new at this. Fixed.

Comment (edm00se) [reply readme.md]: Not a problem.

**Comment by edm00se** [comment]
Thanks for contributing!

**Pull request merged**